site.json

```diff
@@ -38,5 +38,65 @@
       "text": "Security releases now available",
       "link": "https://nodejs.org/en/blog/vulnerability/august-2023-security-releases/"
     }
-  }
+  },
+  "footerLinks": [
+    {
+      "link": "https://openjsf.org/wp-content/uploads/sites/84/2021/01/OpenJS-Foundation-Trademark-Policy-2021-01-12.docx.pdf",
+      "text": "components.footer.links.trademarkPolicy"
+    },
+    {
+      "link": "https://openjsf.org/wp-content/uploads/sites/84/2021/04/OpenJS-Foundation-Privacy-Policy-2019-11-15.pdf",
+      "text": "components.footer.links.privacyPolicy"
+    },
+    {
+      "link": "https://github.com/openjs-foundation/cross-project-council/blob/main/CODE_OF_CONDUCT.md",
+      "text": "components.footer.links.codeOfConduct"
+    },
+    {
+      "link": "https://github.com/nodejs/node/blob/HEAD/SECURITY.md#security",
+      "text": "components.footer.links.security"
+    },
+    {
+      "link": "https://openjsf.org/",
+      "text": "components.footer.links.openJS"
+    }
+  ],
+  "socialLinks": [
+    {
+      "icon": "/static/images/logos/social-github.svg",
+      "link": "https://github.com/nodejs/node",
+      "kind": "dark",
+      "alt": "GitHub"
+    },
+    {
+      "icon": "/static/images/logos/social-github-dark.svg",
+      "link": "https://github.com/nodejs/node",
+      "kind": "light",
+      "alt": "GitHub"
+    },
+    {
+      "icon": "/static/images/logos/social-mastodon.svg",
+      "link": "https://social.lfx.dev/@nodejs",
+      "kind": "neutral",
+      "alt": "Mastodon"
+    },
+    {
+      "icon": "/static/images/logos/social-twitter.svg",
```

Comment:
this logo is wrong in the [storybook](https://64c7d71358830e9105808652-qmpruspquu.chromatic.com/?path=/story/sections-footer--default&globals=theme:dark) despite it being correct at (https://github.com/nodejs/nodejs.org/blob/main/public/static/images/logos/social-twitter.svg

- perhaps this needs a merge from `main`?

Reply:
pulled updates from `main` and rebased